src/allmydata/client.py

```diff
@@ -211,8 +211,12 @@ def init_introducer_clients(self):
         # create a pool of introducer_clients
         self.introducer_clients = []
         for introducer_furl in self.introducer_furls:
-            tubID, location_hints, name = decode_furl(introducer_furl)
-            config_filepath = FilePath(os.path.join(self.basedir, "private", "%s.introduced.yaml" % name))
+            try:
+                tubID, location_hints, name = decode_furl(introducer_furl)
+                config_filepath = FilePath(os.path.join(self.basedir, "private", "%s.introduced.yaml" % name))
+            except ValueError, e:
+                config_filepath = FilePath(os.path.join(self.basedir, "private", "%s.introduced.yaml" % introducer_furl))
```

Comment:
if the FURL is valid then ``decode_furl`` will return without a ValueError exception and in that case we use only he "name" section of the furl in the filename.

if it fails to parse this means it's an invalid furl; this means as a user i can choose to not connect to a specific introducer but instead use the previously cached announcements from that introducer by specifying it's name/swissnum in the "introducer.furl" section of the tahoe.cfg configuration like this:

introducer.furl = introducer